contracts/interfaces/IPoolAAPositionMover.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+/**
+ * @title IPool
+ *
+ * @notice Defines the basic interface for an ParaSpace Pool.
+ **/
+interface IPoolAAPositionMover {
+    function positionMoveToAA(uint256 salt) external returns (address);
```

Comment:
shall we define also events here so that typescript can generate correct typings